src/components/RadioCard/RadioCard.types.ts

```diff
@@ -4,16 +4,19 @@ export type InputEvent = Event
 export type RadioCardProps = {
   checked: boolean
   className?: string
+  content: string | Component
```

Comment:
Do we need to indicate here that `content` is optional?

Reply:
good catch, I did improve the typing in 9f771542